src/build/env.py

```diff
@@ -68,6 +72,19 @@ def _should_use_virtualenv() -> bool:
     )
 
 
+def _subprocess(*args: Any, **kwargs: Any) -> None:
```

Comment:
args needs to be a sequence of str, not? Also for kwargs can we just state other parameters we use?